nutpie/sample.py

```diff
@@ -27,10 +27,12 @@ def sample(
     tune: int = 1000,
     chains: int = 4,
     seed: int = 42,
-    num_try_init=100,
+    num_try_init=200,
     save_warmup: bool = True,
     store_divergences: bool = False,
     progress_bar=True,
+    init_mean=None,
+    store_unconstrained=False,
```

Comment:
Missing from the docstrings

Reply:
fixed, I also added a couple other missing args (from the kwargs)